packages/common/src/store/purchase-content/utils.test.ts

```diff
@@ -170,8 +171,8 @@ describe('store/purchase-content/utils', () => {
     `getBalanceNeeded: $description`,
     ({ amountDue, balance, minPurchaseAmountCents, expected }) => {
       expect(
-        getBalanceNeeded(amountDue, balance, minPurchaseAmountCents)
-      ).toEqual(centsToBN(expected))
+        getBalanceNeeded(amountDue, balance, minPurchaseAmountCents).toNumber()
+      ).toBe(centsToBN(expected).toNumber())
```

Comment:
For some reason this was failing the deep equal check, so changed to check numbers